.eslintrc.json

```diff
@@ -8,12 +8,12 @@
 		"jquery": true
 	},
 	"globals": {
-		"AFCH": true,
-		"Hogan": false,
-		"importScript": false,
-		"mediaWiki": true,
-		"mw": true,
-		"OO": false
+		"AFCH": "writable",
+		"Hogan": "readonly",
+		"importScript": "readonly",
+		"mediaWiki": "writable",
+		"mw": "writable",
```

Comment:
I believe `mw`/`mediaWiki` should be readonly. Is there any code where we actually write into `mw`?